charts/camunda-platform/templates/connectors/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.connectors.enabled -}}
+{{- if .Values.serviceAccount.enabled -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "connectors.serviceAccountName" . }}
+  labels: {{- include "connectors.labels" . | nindent 4 }}
+  {{- with .Values.serviceAccount.annotations }}
```

Comment:
@aabouzaid considering, I commented out line `2`, this one would also fail with

```
Error:  templates/: template: camunda-platform/templates/connectors/templates/service.yaml:11:18: executing "camunda-platform/templates/connectors/templates/service.yaml" at <.Values.service.annotations>: nil pointer evaluating interface {}.annotations
```

Reply:
The same as above, `serviceAccount` is under `connectors`.